monitoring/monitoring/monitoring.py

```diff
@@ -29,6 +34,16 @@
 
 deploy_config = get_deploy_config()
 
+GCP_REGION = os.environ['HAIL_GCP_REGION']
+BATCH_GCP_REGIONS = set(json.loads(os.environ['HAIL_BATCH_GCP_REGIONS']))
+BATCH_GCP_REGIONS.add(GCP_REGION)
+
+PROJECT = os.environ['PROJECT']
+
+DISK_STATES = pc.Gauge('batch_disk_states', 'Batch disk states', ['state', 'namespace', 'zone'])
+DISK_SIZE_GB_SUMMARY = pc.Summary('batch_disk_size_gb', 'Summary of user batch disk sizes (GB)', ['namespace', 'zone'])
+INSTANCE_STATUSES = pc.Gauge('batch_instance_statuses', 'Batch instance statuses', ['status', 'namespace', 'zone', 'machine_type', 'preemptible'])
```

Comment:
can you remove `states` and `statuses` from the metric names? I like that you've made them tags instead of separate metrics for each, but now the name is oddly biased toward a particular tag. Just `batch_disks` and `batch_worker_instances` feels cleaner, and suggests that you can aggregate by tags other than state and status.